<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseFileWindow</class>
 <widget class="QWidget" name="ChooseFileWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open file</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(200, 200, 200);
}
</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: rgb(150, 0, 0);

	border: 1px solid  rgb(100, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: rgb(180, 0, 0);
}

QPushButton:disabled {
	background-color: rgb(200, 100, 100);
}
</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: rgb(120, 120, 120);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}

QPushButton::hover {
	background-color: rgb(150, 150, 150);
}

QPushButton:disabled {
	background-color: rgb(200, 200, 200);
}
</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 11pt &quot;Liberation Sans&quot;;
	color: rgb(20, 20, 20);
}
</string>
   </property>
   <property name="text">
    <string>Choose file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: rgb(0, 150, 0);

	border: 1px solid  rgb(0, 100, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: rgb(0, 180, 0);
}

QPushButton:disabled {
	background-color: rgb(100, 200, 100);
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 10pt &quot;Liberation Mono&quot;;
	color: rgb(20, 20, 20);
	background-color: rgb(230, 230, 230);
}
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>path/to/file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
